### ISSUE TITLE
[Container] Author must reload if page was initially rendered in preview mode

### ISSUE THREAD

**Issue by ky940819** [adobe/aem-core-wcm-components#764]
## Bug Report

**Current Behavior**
When an author loads a page (or reloads a page), and the edit mode is set to PREVIEW, then the "new" placeholder (RT:core/wcm/components/container/v1/container/new) is not rendered.

Subsequently, if the author changes their view to "EDIT" mode, they are not presented with the drop container, because it was never rendered.

The only recourse the author has is to refresh the page after selecting "EDIT" mode.

*Edit*: Just to clarify, while this behaviour occurs on both simple and responsive layout, only simple can be corrected in this project.

**Expected behavior/code**
The author should be able to switch between EDIT and PREVIEW mode seamlessly without having to refresh the page, regardless of which mode the page was initially rendered in.

**Environment**

Component | Version
------------ | -------------
 AEM version | 6.4.5.0
 Core Components version | 2.6.0
 JRE version | OpenJDK Runtime Environment(build 1.8.0_222-b10) 

**Possible Solution**
I believe that this is a simple fix.
in container.html changing the following:
```HTML
<sly data-sly-test="${!isAllowedApplicable && wcmmode.edit}"
             data-sly-resource="${resource.path @ resourceType='core/wcm/components/container/v1/container/new', appendPath='/*', decorationTagName='div', cssClassName='new section'}" />
```
to the following:
```HTML
<sly data-sly-test="${!isAllowedApplicable && (wcmmode.edit || wcmmode.preview)}"
             data-sly-resource="${resource.path @ resourceType='core/wcm/components/container/v1/container/new', appendPath='/*', decorationTagName='div', cssClassName='new section'}" />
```
This should be safe to do, many other components do this, the new container component is display:none inside of a preview pane anyway, so it still does not show on preview (this is the desired behaviour).

I can submit a PR.

**Additional context / Screenshots**
Just for some additional context; the current (incorrect IMO) behavior for the container component is identical as that for the OOTB responsivegrid component (wcm/foundation/components/responsivegrid); this is probably where the current issue came from; however, that is outside the scope of this project - hint hint to anyone at Adobe to fix the issue there.

**Comment by bpauli** [comment]
PR has been merged in https://github.com/adobe/aem-core-wcm-components/commit/cbe64e0d7cd74f3404fcea1003351c9873971707

**Comment by richardhand** [comment]
Verified, merged to master and will be available in the upcoming [2.7.0 release](https://github.com/adobe/aem-core-wcm-components/milestone/17).
